tests/ui/perfherder/alerts_test.jsx

```diff
@@ -664,28 +464,21 @@ describe('"My alerts" checkbox\'s display behaviors', () => {
 
   test('Not displayed in Alerts view (detailed mode)', async () => {
     const { queryByText } = alertsViewControls({
-      isListingAlertSummaries: false,
+      isListMode: false,
     }); // as Django detailed view mode
 
     expect(queryByText('My alerts')).not.toBeInTheDocument();
   });
 
-  test('Not displayed in Alerts view (detailed mode), even when param is missing', async () => {
```

Comment:
Just a question: why this test case was deleted?

Reply:
This does not apply anymore, because I'm passing a prop from AlertsView to know for sure if we are in `ListMode` or `DetailMode` (`isListMode`) and that prop is required, eliminating the edge case the test was covering.

We still have this test above: `Not displayed in Alerts view (detailed mode)`